ci/sync_envoy.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash -e
+
+set -o pipefail
```

Comment:
nit: if you add `set -o errexit` here, it's the same as `bash -e`, except (IMHO) a bit clearer to the reader.

Reply:
Also consider `set -o nounset`, which is also a useful check, though not always what you want.

Reply:
nouunset would not work here if it is checking for an unset var

Reply:
> nit: if you add set -o errexit here, it's the same as bash -e, except (IMHO) a bit clearer to the reader.

thinking on this im not sure - my guess is that you if you surveyed devs more would recognize `-e` as its just far more common

re nounset - i guess it can probably work if i re/set it from the env